packages/storybook/src/nimble/table-column/mapping/table-column-mapping.mdx

```diff
@@ -25,35 +33,51 @@ The <Tag name={tableColumnMappingTag}/> renders specific number, boolean, or str
 <Controls of={tableColumnMappingStories.mappingColumn} />
```

Comment:
I noticed a small bug in the API docs for `validity`.

`unsupportedMappingType: true when the column contains a mapping element other than nimble-mapping-text`

should be something like this

`unsupportedMappingType: true when the column contains a mapping element other than icon, spinner, text, or empty`

Reply:
Fixed in `packages/nimble-components/src/table-column/enum-base/tests/shared-storybook-docs.ts`